xml/System.Net/IPEndPoint.xml

```diff
@@ -568,10 +568,12 @@
         <Parameter Name="s" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A <see cref="T:System.RedOnlySpan`1" /> that contains an IP endpoint in dotted-quad notation or network byte order for IPv4 and in colon-hexadecimal notation for IPv6.</param>
+        <summary>Converts an IP network endpoint (address and port) represented as a <see cref="T:System.RedOnlySpan`1" /> to an IPEndPoint instance.</summary>
+        <returns>An <see cref="T:System.Net.IPEndPoint" /> instance.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Remarks still todo?

Reply:
Since overloads are generated in the same html, for this case and `TryParse` I think it would be redundant to have practically the same remarks twice.

Reply:
Remarks are optional, @scalablecory